site-packages/integralstor_utils/disks.py

```diff
@@ -591,25 +575,43 @@ def get_disk_info_all():
         return return_dict, None
 
 
+def is_mounted(disk_name):
+    """Given a disk name like sda, returns true if its mounted"""
+    try:
+        cmd = "cat /proc/mounts | grep %s" % disk_name
+        output, err = command.get_command_output(
+            cmd, shell=True, check_rc=False)
+        if err:
+            raise Exception(err)
+        elif output == []:
+            is_mounted = False
+        else:
+            is_mounted = True
+    except Exception, e:
+        return None, "Unable to check for device %s: %s" % (disk_name, str(e))
```

Comment:
Please change this comment to print something relevant to the function like "Error checking if the disk is mounted". Otherwise the use will see this on the screen and wonder what you are checking for and what device you are talking about.